src/App.js

```diff
@@ -0,0 +1,31 @@
+var express = require('express')
+var router = express.Router(); //if needed further
+
+var path = require('path');
+
+
+
+const app = express()
+
+app.get('/', function (req, res) {
+    res.write('GET request to the homepage');
+    res.write('Hello Server');
+    res.end();    
+  })
+
+
+app.get('/dirname', function (req, res) {
```

Comment:
I'm not following what this is for? :thinking: 

I think we can have only the `/` endpoint for now.

Reply:
the /dirname will send the directory address (pwd) and all files under the folder